back/app/services/concerns/json_forms_ideas_overrides.rb

```diff
@@ -44,12 +44,47 @@ def custom_form_to_ui_schema fields, locale='en'
           type: 'Category',
           options: { id: 'extra' },
           label: I18n.t('custom_forms.categories.extra.title', locale: locale),
-          elements: fields.reject(&:built_in?).map{|f| yield f}
+          elements: fields.reject(&:built_in?).map{|f| yield f, previousScope = '#/properties/custom_field_values/properties/'}
```

Comment:
I partially answered while you reviewed, but I'll explain better because this is important and it might have a beter solution. 
The custom field values are all saved as jsonb in one column, so the api awaits a custom_field_values param json with all extra fields grouped in there. 
The jsonschema decribes this :
it has the main ideas properties at the root level, and the custom field values nested in the custom_field_values object.
So the jsonschema is something like 
`{
  type: object,
  properties: {
    title_multiloc: { ... }
    ...
    custom_field_values: {
      type: object,
      properties: { **here is the schema for extra fields** }
`
The uiSchema controls need to specify the path of the property the control displays and changes in the jsonschema, so for top-level fields it will be something like : 'properties/${fieldKey}' but for extra fields it needs to be 'properties/custom_fields/properties/${fieldKey}